prizetap/validators.py

```diff
@@ -15,6 +15,7 @@ def __init__(self, *args, **kwargs):
         self.user_profile: UserProfile = kwargs["user_profile"]
         self.raffle: Raffle = kwargs["raffle"]
         self.raffle_data: dict = kwargs.get("raffle_data", dict())
+        self.request = kwargs.get("request", None)
```

Comment:
request is bad name for this variable since it may be mistaken by django request